SU-SWS/stanford_basic
# D8CORE-4246 Inherit font family when using font awesome classes

# READY FOR REVIEW

# Summary
- When using font awesome icons, the FA library sets the font family for the entire element. When that's done on a `<p>` tag, we get unwanted font changes. This inherits the font family when that happens.

# Need Review By (Date)
- 11/19

# Urgency
- medium

# Steps to Test
1. checkout this branch.
2. in a wysiwyg populate with two `<p>` tags with some text.
3. in the first `<p>` tag, add a font awesome icon classes to produce an icon at the front of the text
4. save and verify the two `<p>` tags have the same font family but the icon still displays.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
